search.ipynb

```diff
@@ -734,10 +590,20 @@
   {
    "cell_type": "code",
    "execution_count": 5,
-   "metadata": {
-    "collapsed": true
-   },
-   "outputs": [],
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "ModuleNotFoundError",
+     "evalue": "No module named 'networkx'",
```

Comment:
It seems you have not installed the `networkx` library, and thus you introduced an error here. Install the library and rerun the cell.